conf/mydumper.ini.sample

```diff
@@ -8,7 +8,10 @@ user = root
 # User password
 password = pwd
 # Database to dump
-database = xx
+database = ^(mysql|sys|information_schema|performance_schema)$
+# If valid regex is provided whether or not to invert the match
+# This option should be replaced as soon as a GPL-3.0 compliant go-pcre lib is found
+database_invert_match = on
```

Comment:
Hello,
The `database` and `database_invert_match`  parameters are more confused for the people who don't need this feature.
If we can't find a pure go-pcre lib, we can use the golang's regex to rule the database filter?

Reply:
I do not quite understand. We of course use golangs regex, because it's the only thing available with GPLv3.

What might be a good idea is to split up the config sample in an simple config (remove anything optional and no regex) and an advanced config (with all the optional options and regex).